evalml/tests/automl_tests/test_automl.py

```diff
@@ -1937,11 +1937,11 @@ class Pipeline2(DummyPipeline):
         pipelines_per_batch=5,
         ensembling=False,
         text_in_ensembling=False,
-        pipeline_params=pipeline_parameters,
-        custom_hyperparameters=None,
+        search_parameters=pipeline_parameters,
     )
     automl.automl_algorithm.allowed_pipelines = allowed_pipelines
-    automl._SLEEP_TIME = 0.000001
+    for pipeline in allowed_pipelines:
```

Comment:
is this added in because we skip the new tuner-pipeline logic you added into the algorithms? Don't really see a way around this so I think this is good.

Reply:
Fixed some formatting!